pymeasure/instruments/feeltech/feeltechFY6800.py

```diff
@@ -0,0 +1,122 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+import logging
+from enum import Enum
+from pymeasure.instruments import Instrument, Channel
+from pymeasure.adapters import SerialAdapter
+
+
+from pymeasure.instruments.validators import (
+    strict_discrete_set,
+    truncated_range,
+    strict_range,
+)
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class CustomEnum(Enum):
+    """Provides additional methods to IntEnum:
+    * Conversion to string automatically replaces '_' with ' ' in names
+      and converts to title case
+    * get classmethod to get enum reference with name or integer
+    .. automethod:: __str__
+    """
+
+    def __str__(self):
+        """Gives title case string of enum value."""
+        return str(self.value).replace("_", " ")
+        # str() conversion just because of pylint bug
+
+    @classmethod
+    def get(cls, input_value):
+        """Gives Enum member by specifying name or value.
+        Args:
+            input_value (str|int): Enum name or value
+        Returns:
+            _type_: enum member
+        """
+
+        if isinstance(input_value, int):
+            return cls(input_value)
+        else:
+            return cls[input_value.upper()]
+
+
+class AWGChannel(Channel):
+    """Implementation of a AWG channel."""
+
+    waveform_type = Instrument.control(
+        "R<ch>W",
+        "W<ch>W%s",
```

Comment:
For channels, we use the "f-string" conversion (see `str.format()`), so you should use `{ch}`:
```suggestion
        "R{ch}W",
        "W{ch}W%s",
```

Obviously, this has to reflect in `insert_id` as well.